src/pool/srv_target.c

```diff
@@ -774,8 +775,15 @@ ds_pool_tgt_connect_handler(crt_rpc_t *rpc)
 	hdl->sph_capas = in->tci_capas;
 	hdl->sph_pool = pool;
 
+	rc = daos_iov_copy(&in->tci_cred, &hdl->sph_cred);
```

Comment:
Just realized, the first param of daos_iov_copy is supposed to be the destination. Everything else looks okay to me.